src/zcash/IncrementalMerkleTree.hpp

```diff
@@ -19,7 +19,7 @@ namespace libzcash {
 
 
 typedef uint64_t SubtreeIndex;
-typedef std::array<uint8_t, 32> SubtreeRoot;
+typedef uint256 SubtreeRoot;
```

Comment:
I had thought that we'd decided that it was better to use the array, because we wanted to move away from using uint256 for things that aren't actually numbers?